resources/testharness.js

```diff
@@ -2475,6 +2482,14 @@ policies and contribution forms [3].
                     {
                         clearTimeout(this.timeout_id);
                     }
+                } else if (p == "single_test") {
+                    if (value !== true) {
+                        this.status.status = this.status.ERROR;
+                        this.status.message = "If present, the `single_test` option must be set to `true`.";
```

Comment:
I think we should just go with a truthy value consistent with the above cases. There could possibly be some mistake involving a non-true value that this would catch, but it seems fairly unlikely. Not worth the extra code and tests for it IMHO.

Reply:
The motivation is not so much to reject other truthy values but to reject all falsey ones. I'd have sympathy for the person who assumed that they could write `setup({ single_test: false })` to disable the functionality.

But then again, even though that isn't supported, it's the default behavior, so silently tolerating it wouldn't actually lead to a mismatch between expectation and reality.

I'll remove this.